MODULE.bazel.lock

```diff
@@ -61,7 +61,7 @@
   "moduleExtensions": {
     "//bazel:bzlmod.bzl%setup_mongo_python_toolchains": {
       "general": {
-        "bzlTransitiveDigest": "udzTB+HujjVUA6vSR3OXVyv56PZcC/M/edoFiXCQLsA=",
+        "bzlTransitiveDigest": "ekf0ziDisVXRX3wClarNs1fbz3N5Zahw2icMJsL0OHA=",
```

Comment:
This diff is generated by `bazel` itself. I don't completely understand its purpose but I think it doesn't affect the build result.

Reply:
Here is what google Ai suggested:
> bzlTransitiveDigest is a hash value in the Bazel MODULE.bazel.lock file that represents the inputs to a module extension, ensuring the lockfile correctly reflects the state of external dependencies

So if it is not hallucinating this is just a reflection of the changes in `bazel/utils.bzl`